### DISCUSSION TITLE
Click Button to navigate to external website in a new tab

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#6065] [Help] [answered]
Hi guys

I need a button that navigates to an external website in a new tab.

It appears that if I use "link" element, I can achieve this. 
But if I use "Button" element, define Navigate to action, there is no way to specify new tab?

**Comment by melohagan** [comment, marked as answer]
Hi @magickaito 

It may be possible to add another action "Open Link", or a new checkbox in the "Navigate To" action. 

What would make me pause for the latter, is that it's called "Navigate To", so the implication is that you will be redirected. 

What you could do is update app state: https://docs.budibase.com/docs/app-state
And then display a link on that state being set: https://docs.budibase.com/docs/conditions

**Reply by magickaito** [reply]
Thanks for the confirmation.

**Reply by marian001** [reply]
Ability to use Navigate To action to open a link in new tab would help...   I'd like to add a printer icon for each row of a table, so that a user can open a screen (in new tab/window) with the content ready for printing (e.g. without navigation bar). After printing, the user can close that tab/window and will return to the screen with the table with icons...

Currently, if I want the new tab/window functionality, I can't use the pretty printer icon..

**Comment by Brrrradley** [comment]
> Hi guys
> 
> I need a button that navigates to an external website in a new tab.
> 
> It appears that if I use "link" element, I can achieve this. But if I use "Button" element, define Navigate to action, there is no way to specify new tab?

If you use the javascript option of Navigate To and do `window.open("YourURL","_blank");` it ought to open in a new tab.

**Comment by immohacker** [comment]
Hi,
Thank you,
Exist an variable for the instance?
I would send as parameter an screen but i got an permission error.

my Javascript : window.open("/gutschriftreport","_blank");

Then its open but with error, because app_dev /or pblish id is missing
window.open("{{ appid }}/gutschriftreport","_blank");

Have anybody an idea how i can send an screen?
This would be nice, so i can use the navigate to as an new tab with screen
Thank you!
Best regards Bonkersdeluxe
